solr/webapp/web/partials/query.html

```diff
@@ -359,6 +362,10 @@
         </div>
       </div>
 
+      <label for="json_query" title="JSON Request API">
+        JSON Query (<a href="https://solr.apache.org/guide/json-request-api.html" target="_blank">DSL documentation</a>)
```

Comment:
what about just having a icon for the link to the docs...   One is "Json Facet DSL", the other is "DSL documentation"

Reply:
Good idea. Do you know how to do it with a CSS class instead img tag?

Reply:
Ok, I made it, looks quite good:
<img width="340" alt="Skjermbilde 2022-01-29 kl  13 32 53" src="https://user-images.githubusercontent.com/409128/151661149-25914bd3-1918-46d8-b1df-ed4bdd7d6e35.png">
I'd wish we had an "arrow" or "chain" icon that would more clearly suggest that a new browser window will open.

Reply:
The icon "book-open-text.png" shows an open book and is perhaps an alternative, suggesting a documentation link?

![book-open-text](https://user-images.githubusercontent.com/409128/151661293-df6874d4-6565-40e8-8297-16c57c6a55af.png)

Reply:
I would introduce the book only as part of a pass through all the various screens.